rockcraft/lifecycle.py

```diff
@@ -39,6 +39,24 @@ def pack():
         pack_in_provider(project)
         return
 
+    for platform_entry, platform in project.platforms.items():
```

Comment:
Hi! What happens if `project.platforms` is empty? Or more specifically: is this new primitive mandatory?

Reply:
hey!

indeed it is mandatory. it cannot be empty. The project expected a Dict to be passed (https://github.com/canonical/rockcraft/pull/52/files#diff-0ea7d1a44d76c7792a852505490ca9e3fb738ec2a7814b051ebc828efb8ceceeR148)

Reply:
If it's mandatory, can you please the tutorial in `doc/tutorials.rst`? It's short but I believe both the `rockcraft.yaml` sample and the archive name are incorrect now.

Either that or we should add a task to update the tutorial soon-ish (I would also be fine with this)

Reply:
y missed that one.
fixed now: https://github.com/canonical/rockcraft/pull/52/commits/58db8095731dd627a27224180a84afe77fae27b1

thanks for spotting it.

This should actually be automated. The doc's tutorial should fetch the YAML from https://github.com/canonical/rockcraft/pull/52/files#diff-9acc5f73b478b10cf158341e7a1ba82274b3912ed6f9e040fd04372acf75abdc, which is actually being tested by the CI. Otherwise, we start having duplicated and untested content in the tutorials. This is a good issue to have in the backlog: https://github.com/canonical/rockcraft/issues/54